src/Microsoft.Azure.WebJobs.Logging.ApplicationInsights/README.md

```diff
@@ -0,0 +1,49 @@
+﻿# Microsoft.Azure.WebJobs.Logging.ApplicationInsights
+
+This package adds Application Insights based logging capabilities to the WebJobs SDK. For more information, please visit https://learn.microsoft.com/en-us/azure/app-service/webjobs-sdk-get-started.
+
+## Commonly used types
+
+- `ApplicationInsightsLoggingBuilderExtensions`
+- `ApplicationInsightsLoggerProvider`
+- `ApplicationInsightsLoggerOptions`
+
+## Example usage
+
+The below example demonstrates configuration of ApplicationInsights logging on host startup via the `AddApplicationInsightsWebJobs` builder method.
+
+``` CSharp
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Logging;
+
+class Program
+{
+    public static async Task Main(string[] args)
+    {
+        var builder = new HostBuilder()
```

Comment:
Recommend same builder changes here.

Reply:
Applied some of your changes, however the logger config code needs to remain.

Reply:
Logger config is fine to remain. The important part to change is switching from `new HostBuilder()` to `Host.CreateDefaultBuilder(args)` to encourage best practices.

Can you make that change here as well as in src/Microsoft.Azure.WebJobs.Host.Storage/README.md?

Reply:
Got it. Fixed all the places.